app/Http/Controllers/WorkspaceController.php

```diff
@@ -31,6 +34,87 @@ public function listUsers(Request $request, $workspaceId)
         return (new WorkspaceHelper($workspace))->getAllUsers();
     }
 
+    public function addUser(Request $request, $workspaceId)
```

Comment:
The specs mention that all of these new methods should go into a new dedicated `WorkspaceUserController`